ccd-definition/ComplexTypes/Recipients.json

```diff
@@ -0,0 +1,93 @@
+[
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "Recipients",
+    "ListElementCode": "name",
+    "FieldType": "Text",
+    "ElementLabel": "Name of recipient",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "Recipients",
+    "ListElementCode": "addressCheck",
+    "FieldType": "YesOrNo",
+    "ElementLabel": "Do you have the recipient's address?",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "Recipients",
+    "ListElementCode": "address",
+    "FieldType": "AddressUK",
+    "ElementLabel": "Recipient's address",
+    "HintText": "Enter a UK postcode",
+    "FieldShowCondition": "addressCheck=\"Yes\"",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "Recipients",
+    "ListElementCode": "documents",
```

Comment:
Can we be more specific with the naming here?

Reply:
Discussed and keeping as is as the text box description is very generic and has the ability to type anything about the documents.